types/node/v8/index.d.ts

```diff
@@ -4900,7 +4889,7 @@ declare module "tls" {
          * @param callback - callback(err) will be executed with null as err, once the renegotiation
          * is successfully completed.
          */
-        renegotiate(options: TlsOptions, callback: (err: Error | null) => void): any;
+        renegotiate(options: { rejectUnauthorized: boolean, requestCert: boolean }, callback: (err: Error | null) => void): any;
```

Comment:
`rejectUnauthorized` and `requestCert` are. optional

Reply:
Thanks, that could have been bad!